spec/javascript/components/lease_ups/LeaseUpApplicationsPage.test.js

```diff
@@ -179,5 +179,25 @@ describe('LeaseUpApplicationsPage', () => {
       // Expect alert to be closed
       expect(wrapper.find('.alert-body').exists()).toBe(false)
     })
+
+    test('Should open closeable alert modal on failed submit', async () => {
```

Comment:
looks like you might have just wiped out your updated test. This one is a duplicate of above.

Reply:
yep, replaced and pushed, I must have missed a commit in my rebase